id-to-url.js

```diff
@@ -1,14 +1,37 @@
 const DEFAULT_PORT = require('./port');
 
+const base = `http://localhost:${DEFAULT_PORT}`
+
 module.exports = function idToUrl(blobId, params) {
-  const port = (params && params.port) || DEFAULT_PORT;
+  const url = new URL(base)
+
   const [pureBlobId, query] = blobId.split('?');
-  const blobRef = encodeURIComponent(pureBlobId);
-  const paramsStr = query
-    ? '?' + query
-    : params && params.unbox
-    ? `?unbox=${encodeURIComponent(params.unbox.toString('base64'))}.boxs`
-    : '';
-
-  return `http://localhost:${port}/get/${blobRef}${paramsStr}`;
+
+  if (params && params.hostname) url.hostname = params.hostname;
```

Comment:
```suggestion
  if (params?.hostname) url.hostname = params.hostname;
```

Reply:
sure thing, was just following pattern of this module, in case we were suporting old node